crates/burn-hip/src/lib.rs

```diff
@@ -18,13 +18,15 @@ pub type Hip<F = f32, I = i32> = JitBackend<HipRuntime, F, I>;
 #[cfg(feature = "fusion")]
 pub type Hip<F = f32, I = i32> = burn_fusion::Fusion<JitBackend<HipRuntime, F, I>>;
 
-#[cfg(target_os = "linux")]
-#[cfg(test)]
-mod tests {
-    use burn_jit::JitBackend;
-
-    pub type TestRuntime = cubecl::hip::HipRuntime;
-    pub use half::{bf16, f16};
-
-    burn_jit::testgen_all!();
-}
+// TODO: Hang the computer when AMD isn't available.
+//
+// #[cfg(target_os = "linux")]
+// #[cfg(test)]
+// mod tests {
+//     use burn_jit::JitBackend;
+//
+//     pub type TestRuntime = cubecl::hip::HipRuntime;
+//     pub use half::{bf16, f16};
+//
+//     burn_jit::testgen_all!();
+// }
```

Comment:
Not sure if this is meant to be commented out just locally or actually committed to fix some HIP stuff?

Reply:
It breaks rust-analyzer, so let's comment them until it's fixed.